src/battle_anim.c

```diff
@@ -676,7 +681,9 @@ static void Cmd_createvisualtaskontargets(void)
     }
 
     numArgs = GetBattleAnimMoveTargets(battlerArgIndex, targets);
-
+    if (numArgs == 0)
+        return;
```

Comment:
return with no value

Reply:
both functions have a void return type though?

Reply:
You're right, I looked at the wrong function sorry